pkg/task/common.go

```diff
@@ -178,18 +201,40 @@ func (cfg *Config) ParseFromFlags(flags *pflag.FlagSet) error {
 }
 
 // newMgr creates a new mgr at the given PD address.
-func newMgr(ctx context.Context, pds []string) (*conn.Mgr, error) {
+func newMgr(ctx context.Context, pds []string, tlsConfig TLSConfig) (*conn.Mgr, error) {
+	var (
+		tlsConf *tls.Config
+		err     error
+	)
 	pdAddress := strings.Join(pds, ",")
 	if len(pdAddress) == 0 {
 		return nil, errors.New("pd address can not be empty")
 	}
 
+	securityOption := pd.SecurityOption{}
+	if tlsConfig.Enable() {
+		conf := config.GetGlobalConfig()
+		conf.Security.ClusterSSLCA = tlsConfig.CA
+		conf.Security.ClusterSSLCert = tlsConfig.Cert
+		conf.Security.ClusterSSLKey = tlsConfig.Key
+		config.StoreGlobalConfig(conf)
```

Comment:
please put these in the glue, we don't need to touch TiDB's own global config when integrating BR into TiDB.